packages/nc-gui-v2/components/cell/Rating.vue

```diff
@@ -35,12 +35,13 @@ const localState = computed({
   <div class="d-100 h-100" :class="{ 'nc-cell-hover-show': localState === 0 || !localState }">
     <v-rating v-model="localState" :length="ratingMeta.max" dense x-small :readonly="readOnly" clearable>
       <template #item="{ isFilled, click }">
-        <v-icon v-if="isFilled" :size="15" :color="ratingMeta.color" @click="click">
-          <MdiStarIcon />
-        </v-icon>
-        <v-icon v-else :color="ratingMeta.color" :size="15" class="nc-cell-hover-show" @click="click">
-          <MdiStarOutlineIcon />
-        </v-icon>
+        <!--        todo : custom color and icon -->
+        <!--        <v-icon v-if="isFilled"- :size="15" :color="ratingMeta.color" @click="click"> -->
+        <MdiStarIcon v-if="isFilled" @click="click" />
```

Comment:
To apply color to an icon you can use windicss -> `<MdiStarIcon class="text-primary" />`

Or in this case probably something like `<MdiStarIcon :class="`text-[${ratingMeta.color}]` />